src/isomorphic/children/__tests__/onlyChild-test.js

```diff
@@ -13,17 +13,15 @@
 
 describe('onlyChild', () => {
   var React;
-  var onlyChild;
   var WrapComponent;
 
   beforeEach(() => {
     React = require('react');
```

Comment:
@gaearon How does this `require` work right now? I couldn't find any jest mocks for `'react'`